virtual_rainforest/models/litter/litter_pools.py

```diff
@@ -38,12 +34,15 @@ def calculate_litter_pool_updates(
     woody: NDArray[np.float32],
     below_metabolic: NDArray[np.float32],
     below_structural: NDArray[np.float32],
+    lignin_above_structural: NDArray[np.float32],
+    lignin_woody: NDArray[np.float32],
+    lignin_below_structural: NDArray[np.float32],
     decomposed_excrement: NDArray[np.float32],
     decomposed_carcasses: NDArray[np.float32],
     update_interval: float,
     constants: LitterConsts,
 ) -> dict[str, DataArray]:
```

Comment:
I find a bit odd that this function takes as inputs only numpy arrays but returns DataArrays. It means it implicitly knows that the calling function expects DataArrays with a dimension name "cell_id", when it should be the other way around.

I think that, considering this is a top level function that update the input data - kind of - take DataArrays as inputs or return numpy arrays and let the caller to transform them into whatever it needs.